p2p_wallet/Scenes/Main/Main.ViewController.swift

```diff
@@ -42,18 +44,29 @@ extension Main {
 
         override func setUp() {
             super.setUp()
-            add(child: TabBarVC())
+            add(child: tabBar)
             view.addSubview(blurEffectView)
             blurEffectView.autoPinEdgesToSuperviewEdges()
         }
 
         override func bind() {
             super.bind()
+
+            rx.viewWillAppear
+                .take(1)
+                .mapToVoid()
+                .bind(to: viewModel.viewDidLoad)
```

Comment:
seems like a wrong name. Did you mean viewModel.viewWillAppear

Reply:
yes, because rx.viewDidLoad works wrong with binding anywhere